packages/vuetify/src/components/VSwitch/VSwitch.sass

```diff
@@ -33,7 +37,7 @@
 .v-switch__thumb
   align-items: center
   border-radius: $switch-thumb-radius
-  color: rgb(var(--v-theme-surface))
```

Comment:
this SASS conversion was redundant, and also was breaking Cypress component testing.

Not experienced enough with `.sass` files, I'd be curious since `rgb` is a compile-time function, and `var(--v-theme-surface)` is a runtime value, how the conversion is handled?

Nevertheless, the conversion to RGB happens elsewhere and the unit test is going to lock in that behaviour to avoid any breaking change.

Reply:
Not sure how it was breaking cypress, but `rgb(var(--v-theme-surface))` is correct.

Reply:
@nekosaur Yes, the issue not related to Cypress. If you add the below code to the Playground and run `yarn dev`

```vue
<template>

  <div id="parent" :style="{ color: 'rgb(5,45,5)', '--v-theme-surface': 'rgb(200, 200, 200)' }">
    <v-switch />
  </div>

</template>
```

You will notice that the `color` property of the thumb fails to correctly parse the `--v-theme-surface` when wrapped in the `rgb` function. If you remove `rgb` the color is computed correctly.

Any idea?

Reply:
Again, to me make sense that it doesn't work correctly because the value of `--v-theme-surface` will only be known at browser-render time, while the `rgb` will be invoked during the compilation of `node-sass`, so the value cannot be known/parsed ahead of time.

Is there something I am missing?